src/main/fc/runtime_config.h

```diff
@@ -103,6 +105,8 @@ extern uint16_t flightModeFlags;
    [BOXPASSTHRU]    = LOG2(PASSTHRU_MODE),               \
    [BOXFAILSAFE]    = LOG2(FAILSAFE_MODE),               \
    [BOXGPSRESCUE]   = LOG2(GPS_RESCUE_MODE),             \
+   [BOXAUTOLAND]    = LOG2(AUTO_LAND),                   \
```

Comment:
What should happen if both `AUTO_LAND` and `DROP` are active at the same time?

Reply:
Good catch. I'd suggest using the safer of the two, DROP (which kills motors, right)?